FlatGrid.js

```diff
@@ -27,6 +27,7 @@ const FlatGrid = memo(
       invertedRow,
       maxItemsPerRow,
       adjustGridToStyles,
+      customInterface: CustomInterface,
```

Comment:
I think we can call this `customFlatList` to be clear this is to overrider the `FlatList` we use internally. Though the prop would be different for `SectionGrid` and `FlatGrid`. Let me know your thoughts.

Either way, this line can be changed to something like `customFlatList: FlatListComponent = FlatList,` so the default value of FlatList applied. Also I think its better to call the variable `FlatListComponent` (or something similar).

We should also add this to the propTypes at the end of the file.

Reply:
Perfect will do! Appreciate the feedback :)